magento/magento2
# Resolve "Disable Automatic Group Change Based on VAT ID" in "New Customer" form at Backend is not respect Setting "Default Value for Disable Automatic Group Changes Based on VAT ID"

### Description (*)


"Disable Automatic Group Change Based on VAT ID" in "New Customer" form at Backend is not respect Setting "Default Value for Disable Automatic Group Changes Based on VAT ID"

- Go to backend, Store->Configurations
- Customer->Customer Configuration
- In "Create New Account Options", "Default Value for Disable Automatic Group Changes Based on VAT ID" is Yes

![image](https://user-images.githubusercontent.com/8234209/67455808-efaa9680-f658-11e9-9026-475d4bfb3b3e.png)

- Go to New Customer form
- Check "Disable Automatic Group Change Based on VAT ID" should be "**CHECKED**" . Now, it is "**UNCHECKED**"

### Fixed Issues (if relevant)


### Manual testing scenarios (*)

- Go to backend, Store->Configurations
- Customer->Customer Configuration
- In "Create New Account Options", "Default Value for Disable Automatic Group Changes Based on VAT ID" is Yes

![image](https://user-images.githubusercontent.com/8234209/67455808-efaa9680-f658-11e9-9026-475d4bfb3b3e.png)

- Go to New Customer form
- Check "Disable Automatic Group Change Based on VAT ID" 

![image](https://user-images.githubusercontent.com/8234209/67455710-a9edce00-f658-11e9-8057-cc97c52825d6.png)

### Questions or comments


### Contribution checklist (*)
